ckan/lib/formatters.py

```diff
@@ -137,7 +137,7 @@ def months_between(date1, date2):
         'day': datetime_.day,
         'year': datetime_.year,
         'month': _MONTH_FUNCTIONS[datetime_.month - 1](),
-        'timezone': datetime_.tzinfo.zone,
+        'timezone': datetime_.tzinfo._tzname,
```

Comment:
> (I know this is out of the scope of this PR)
I know :)

What about the private property? Is there a public method on datetime / tzinfo that does something similar?